Directory.Build.targets

```diff
@@ -16,6 +16,7 @@
 
   <Import
       Project="$([System.IO.Path]::GetDirectoryName($(MSBuildThisFileDirectory))).override.targets"
-      Condition=" Exists('$([System.IO.Path]::GetDirectoryName($(MSBuildThisFileDirectory))).override.targets') "
-  />
+      Condition=" Exists('$([System.IO.Path]::GetDirectoryName($(MSBuildThisFileDirectory))).override.targets') "/>
+
+  <Import Project="RedundantSigningWorkarounds.targets" />
```

Comment:
If choose to go this route I think we should only import this in the projects that declare `@(FilesToSign)` items, we don't need every project in the repo to import this.